lxd/main_init.go

```diff
@@ -40,6 +41,40 @@ type CmdInit struct {
 
 // Run triggers the execution of the init command.
 func (cmd *CmdInit) Run() error {
+	// Check that command line arguments don't conflict with each other
+	err := cmd.validateArgs()
+	if err != nil {
+		return err
+	}
+
+	// Connect to LXD
+	client, err := lxd.ConnectLXDUnix(cmd.SocketPath, nil)
+	if err != nil {
+		return fmt.Errorf("Unable to talk to LXD: %s", err)
+	}
+
+	// Kick off the appropriate run mode (either preseed, auto or interactive).
+	if cmd.Args.Preseed {
+		err = cmd.runPreseed(client)
+	} else {
+		err = cmd.runAutoOrInteractive(client)
+	}
+
+	if err == nil {
+		cmd.Context.Output("LXD has been successfully configured.\n")
+	}
+
+	return err
+}
+
+// Run the logic for auto or interactive mode.
+//
+// IMPLEMENTATION NOTE FOR REVIEWERS (free): this logic is going to be
```

Comment:
We stay away from adding our names to comments as this gets easily confusing. When we're interested we query `git blame`. :)